src/transcript-plugin.tsx

```diff
@@ -47,6 +49,7 @@ const logger = getContribLogger({
   module: "transcript-plugin"
 });
 
+const { get } = ObjectUtils;
```

Comment:
why import all ObjectUtils and then grab the 'get' only? 
Why not import the get directly from ObjectUtils ?